executor/replace.go

```diff
@@ -89,6 +91,33 @@ func (e *ReplaceExec) removeRow(ctx context.Context, txn kv.Transaction, handle
 	return false, nil
 }
 
+// EqualDatums compare if a and b contains the same datum values.
+func (e *ReplaceExec) EqualDatums(sc *stmtctx.StatementContext, a []types.Datum, b []types.Datum) (bool, error) {
```

Comment:
How about renaming it to, say, like `EqualDatumsIgnoreCollation` or `EqualDatumsAsBinary`, and update the comment, so that it is more straightforward to the other developers?

Reply:
I have renamed it to `EqualDatumsAsBinary`.